ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -92,16 +95,16 @@ public struct Avatar: View, ConfigurableTokenizedControl {
         state.primaryText = primaryText
         state.secondaryText = secondaryText
 
-        self.state = state
+        self.init(state)
```

Comment:
The documentation comment for this init says it should be used for "internal container views". Should we be using it here, or should we update the comment to just say "internal" and leave out the "container view"?